scripts/zkai_attention_kv_softmax_paired_source_validation_audit_gate.py

```
@@ -0,0 +1,287 @@
+def run_sidecar_target(target: Target) -> dict[str, Any]:
+    module = target.module
+    source_input = module.read_bounded_json(module.SOURCE_INPUT_JSON, module.MAX_SOURCE_INPUT_JSON_BYTES, "source input")
+    envelope = module.read_bounded_json(
+        module.LOOKUP_ENVELOPE_JSON, module.MAX_LOOKUP_ENVELOPE_JSON_BYTES, "lookup envelope"
+    )
+    mutated_source = mutate_source(source_input)
+    mutated_envelope = copy.deepcopy(envelope)
+    mutated_envelope["source_input"] = mutated_source
+    try:
+        module.validate_lookup_envelope(mutated_envelope, mutated_source, module.LOOKUP_ENVELOPE_SIZE_BYTES)
+    except Exception as err:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)